src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -77,4 +77,51 @@ public function committeeAutocompleteAction(Request $request)
 
         return new JsonResponse($result ?? []);
     }
+
+    /**
+     * @Route("/mon-comite-soutien/{slug}", name="app_citizen_project_committee_support")
+     * @Method("GET")
+     */
+    public function committeeSupportAction(CitizenProject $citizenProject): Response
+    {
+        if (!$this->getUser()->isSupervisor()) {
+            throw $this->createAccessDeniedException();
+        }
+
+        $committeeManager = $this->get('app.committee.manager');
+        $flashMessage = sprintf('Le projet citoyen %s n\'a pas demandé votre soutient', $citizenProject->getName());
+
+        foreach ($citizenProject->getCommitteeSupports() as $committeeSupport) {
```

Comment:
Too much work performed in this controller. Try to move the business logic to the entity itself and/or manager/voter depending on the context.